src/containers/App.js

```diff
@@ -149,7 +154,10 @@ export default class App extends React.Component {
         }}
         path={path}
         search={doSearch}
-        opened={leftNavIsOpen}/>}
+        opened={leftNavIsOpen}
+        networkNavAnimation={moveWithMenu}
```

Comment:
to have it behave like it's part of topNav I'm ending up duplicating way more css and now looking at adding the scroll listener stuff from topnav also . The code is way simpler if I  just pass the animation parameters.